app/frontend/wallet/blockchain-explorer.ts

```diff
@@ -87,26 +111,28 @@ const blockchainExplorer = (ADALITE_CONFIG) => {
     return Object.values(transactions).sort((a, b) => b.ctbTimeIssued - a.ctbTimeIssued)
   }
 
-  async function fetchTxInfo(txHash) {
+  async function fetchTxInfo(txHash: string) {
     const url = `${ADALITE_CONFIG.ADALITE_BLOCKCHAIN_EXPLORER_URL}/api/txs/summary/${txHash}`
-    const response = await request(url)
-
+    const response: TxSummaryResponse = await request(url)
+    // @ts-ignore (TODO, handle 'Left')
     return response.Right
   }
 
+  // TODO: delete, raw txs are no longer in the db sync database
+  // TODO: remove tests for byron which use this
   async function fetchTxRaw(txId) {
     // eslint-disable-next-line no-undef
     const url = `${ADALITE_CONFIG.ADALITE_BLOCKCHAIN_EXPLORER_URL}/api/txs/raw/${txId}`
     const result = await request(url)
     return Buffer.from(result.Right, 'hex')
   }
 
-  async function isSomeAddressUsed(addresses) {
-    return (await getAddressInfos(addresses)).caTxNum > 0
+  async function isSomeAddressUsed(addresses: Array<string>): Promise<boolean> {
+    return (await _getAddressInfos(addresses)).caTxNum > 0
   }
 
   // TODO: we should have an endpoint for this
-  async function filterUsedAddresses(addresses: Array<String>) {
+  async function filterUsedAddresses(addresses: Array<string>) {
     const txHistory = await getTxHistory(addresses)
     const usedAddresses = new Set()
```

Comment:
This is currently returning a `Set<unknown>` 
```suggestion
    const usedAddresses = new Set<string>()
```